src/Downloader/Args.cs

```diff
@@ -0,0 +1,226 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+
+public struct Args
+{
+    public string Org { get; set; }
+    public List<string> Repos { get; set; }
+    public string GithubToken { get; set; }
+    public string? IssueDataPath { get; set; }
+    public int? IssueLimit { get; set; }
+    public string? PullDataPath { get; set; }
+    public int? PullLimit { get; set; }
+    public int? PageSize { get; set; }
+    public int? PageLimit { get; set; }
+    public int[] Retries { get; set; }
+    public Predicate<string> LabelPredicate { get; set; }
+    public bool Verbose { get; set; }
```

Comment:
I was thinking of maybe trying to unify the args structs, most of them dup quite a bit.

Reply:
I think we've squeezed as much out of the arg factoring as is worthwhile. Once the next version of System.CommandLine is published publicly, I'll likely want to just adopt that into here next time we touch it.